internal/rpchelp/helpdescs_en_US.go

```diff
@@ -0,0 +1,399 @@
+	"getaccountaddress--synopsis": "Returns the most recent unused external payment address for an account.\n" +
+		"A new address is generated for the account if the most recently-generated address has been seen on the blockchain on in mempool.",
+	"getaccountaddress-account":  "The account of the returned address",
+	"getaccountaddress--result0": "The unused address for 'account'",
+
+	// GetAddressesByAccountCmd help.
+	"getaddressesbyaccount--synopsis": "Returns all addresses strings controlled by a single account.",
+	"getaddressesbyaccount-account":   "Account name to fetch addresses for",
+	"getaddressesbyaccount--result0":  "All addresses controlled by 'account'",
+
+	// GetBalanceCmd help.
+	"getbalance--synopsis":   "Calculates and returns the balance of one or all accounts.",
+	"getbalance-minconf":     "Minimum number of block confirmations required before an unspent output's value is included in the balance",
+	"getbalance-account":     "The account name to query the balance for, or '*' to consider all accounts",
+	"getbalance--condition0": "account!=*",
+	"getbalance--condition1": "account=*",
+	"getbalance--result0":    "The balance of 'account' valued in bitcoin",
+	"getbalance--result1":    "The balance of all accounts valued in bitcoin",
+
+	// GetBestBlockHashCmd help.
+	"getbestblockhash--synopsis": "Returns the hash of the newest block in the best chain that wallet has finished syncing with.",
+	"getbestblockhash--result0":  "The hash of the most recent synced-to block",
+
+	// GetBlockCountCmd help.
+	"getblockcount--synopsis": "Returns the blockchain height of the newest block in the best chain that wallet has finished syncing with.",
+	"getblockcount--result0":  "The blockchain height of the most recent synced-to block",
+
+	// GetInfoCmd help.
+	"getinfo--synopsis": "Returns a JSON object containing various state info.",
+
+	// InfoWalletResult help.
+	"infowalletresult-version":         "The version of the server",
+	"infowalletresult-protocolversion": "The latest supported protocol version",
+	"infowalletresult-blocks":          "The number of blocks processed",
+	"infowalletresult-timeoffset":      "The time offset",
+	"infowalletresult-connections":     "The number of connected peers",
+	"infowalletresult-proxy":           "The proxy used by the server",
+	"infowalletresult-difficulty":      "The current target difficulty",
+	"infowalletresult-testnet":         "Whether or not server is using testnet",
+	"infowalletresult-relayfee":        "The minimum relay fee for non-free transactions in BTC/KB",
+	"infowalletresult-errors":          "Any current errors",
+	"infowalletresult-paytxfee":        "The increment used each time more fee is required for an authored transaction",
+	"infowalletresult-balance":         "The balance of all accounts calculated with one block confirmation",
+	"infowalletresult-walletversion":   "The version of the address manager database",
+	"infowalletresult-unlocked_until":  "Unset",
```

Comment:
the problem is that the field is still in the json object, not missing, so the help language should indicate that the value is bogus.